client/talk-plugin-flags/components/FlagButton.js

```diff
@@ -99,6 +99,7 @@ export default class FlagButton extends Component {
             }
           })
           .catch((err) => {
+            this.props.notify('error', t('error.COMMENT_FLAG_FAILED'));
```

Comment:
Can you change these lines to `this.props.notify('error', getErrorMessages(err));` and add an `import {getErrorMessages} from 'coral-framework/utils';` at the top?

Reply:
Sure, will do